src/lib/components/ILSFooter/ILSFooter.js

```diff
@@ -16,10 +16,21 @@ const FooterLayout = props => {
                 <Header as="h4" content="More information" />
                 <List>
                   <List.Item>
-                    <Link to={getStaticPageByName('about').route}>About</Link>
+                    <Link
+                      to={
+                        getStaticPageByName(uiConfig.staticPages, 'about').route
```

Comment:
I see we are passing `uiConfig.staticPages` every time we call the explicit functions `getStaticPageByName`.
What is the case for that? I think it should be `getStaticPagesByName('about')`?

*same comment for the rest of `getStaticPage*`

Reply:
the approach you are proposing didn't work because of the mutated config problem, it was getting the ILS static pages here instead of CDS ones

Reply:
I will handle it in my upcoming pr 👍